src/data/ACTIONS/root/VPR.ts

```diff
@@ -665,20 +667,23 @@ export const VPR = ensureActions({
 		name: 'Twinfang Bite',
 		icon: iconUrl(3731),
 		cooldown: 1000,
+		statusesApplied: ['SWIFTSKINS_VENOM'],
```

Comment:
As long as this won't result in falsely making the modules think you got the buff if you go out of order, I don't see the harm in defining this, but I originally left them out due to me being unsure of how statusesApplied would affect it.